agent/bench-scripts/driver/linpack

```diff
@@ -225,16 +231,22 @@ while true; do
 	esac
 done
 
-binary=${1}
-if [[ -z "${binary}" ]]; then
-	printf -- "[%s] ERROR: You must specify the location of the LINPACK binary\n\n" "${script_name}" >&2
+ver=${1}
+if [[ -z "${ver}" ]]; then
+	printf -- "[%s] ERROR: You must specify the version of the LINPACK binary\n\n" "${script_name}" >&2
 	usage >&2
 	exit 2
 fi
+binary="/usr/local/pbench-linpack-${ver}/benchmarks/linpack/xlinpack_xeon64"
```

Comment:
Just a thought....  How about changing line 14 to
```
binary_template="/usr/local/pbench-linpack-%s/benchmarks/linpack/xlinpack_xeon64"
```
and changing this line to
```
binary=$(printf -- ${binary_template} ${ver})
```
(since I know that you like `printf` 😉).

Alternately, we could set the template like
```
binary_template="/usr/local/pbench-linpack-\${ver}/benchmarks/linpack/xlinpack_xeon64"
```
and change this line to
```
eval binary=${binary_template}
```
but that feels like it might be a little too arcane (and slightly fragile), to me.

Reply:
With the addition of the install-directory parameter, this is largely obviated.